lib/check/do_end_single_expression.ex

```diff
@@ -1,4 +1,4 @@
-defmodule CompassCredoPlugin.Check.DoSingleExpression do
+defmodule CompassCredoPlugin.Check.DoEndSingleExpression do
```

Comment:
hi @liamstevens111 

How about renaming it to `MethodSingleExpression` and the module below from `DoMultipleLines` to `MethodSingleExpressionMultiline`? Maybe you can choose a better name. 

The reason is with the existing names, if we put it in the list of checks, it's a bit hard to understand what it does. As you can see in the list of `.credo.exs`, the name is quite clear likes check-def-delegate-order, check-repeating-fragments, check-single-module file

```
        {CompassCredoPlugin.Check.DefdelegateOrder, []},
        {CompassCredoPlugin.Check.DoSingleExpression, []},
        {CompassCredoPlugin.Check.DoEndSingleExpression, []},
        {CompassCredoPlugin.Check.DoMultipleLines, []},
        {CompassCredoPlugin.Check.RepeatingFragments, []},
        {CompassCredoPlugin.Check.SingleModuleFile, []},
```

Reply:
We can't use `Method` because it doesn't only check `def` and `defp` but also `if` and `unless`. I think the only term that accurately describes them all would be `macro` 😆 . But yeah, to the people outside of existing knowledge of these checks won't have any idea what they mean 👍 

I will have a think but not sure at the moment.

Reply:
What about

`DoEndBlockContainsSingleLine`
`DoBlockContainsMultipleLines`

Might just be confusion on just `DoBlock`

@byhbt

Reply:
`DoEndBlockContainsSingleLine` 👍 

- Should we apply this pattern for the name `DoMultipleLines` => `DoEndBlockContainsMultipleLines`?
- Should we replace `Contains`  by `Has`, `has` is shorter but `contains` seem more expressive?

Reply:
@byhbt Yeah I like `has` better, also removes the `sS` when using contains in the check name. Also gives context and looks better than if we just completed removed the possessive word 👍 

```elixir
DoEndBlockHasSingleLine
DoBlockHasMultipleLines
```

Reply:
that's great! Thanks 🙏

Reply:
Renamed both here [7f8abd6](https://github.com/nimblehq/compass-credo-plugin/pull/27/commits/7f8abd6495bbf9704d9a714d68079f32283f3685)